src/storage/v2/property_store.hpp

```diff
@@ -19,6 +20,42 @@
 
 namespace memgraph::storage {
 
+struct SearchResult {
+  std::uint64_t id;
+  nlohmann::json document;
+  double score;
+};
+
+class ExternalStoreMock {
+ private:
+  std::string INDEX = "";  // placeholder (Tantivy supports multiple indices)
+  std::map<std::string, std::map<std::uint64_t, nlohmann::json>> storage{};  // index_name: {document_id: document}
+
+ public:
+  enum class Consistency : uint8_t { DEFAULT };
+
+  void CreateAllPropsIndex(std::string index_name, LabelId label, std::string tokenizer = "DUMMY_TOKENIZER",
```

Comment:
we should probably add mappings here https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html research required

Reply:
what about the underlying Tantivy config?

Reply:
January 2 meeting: mappings shouldn’t be user-defined here https://www.notion.so/memgraph/Hybrid-Text-Search-Feature-Spec-16fbf5b8678f41ebbe85c81047048f6b?pvs=4#3e637591d3734403aaccfdfd67641546